specification/security/resource-manager/Microsoft.Security/stable/2022-03-01/examples/Pricings/ListPricings_example.json

```diff
@@ -0,0 +1,123 @@
+{
+  "parameters": {
+    "api-version": "2022-03-01",
+    "subscriptionId": "20ff7fc3-e762-44dd-bd96-b71116dcdc23"
```

Comment:
please update 2018 api version with example containg all enums as well

Reply:
good idea :)
I'll create a PR for it